runtime/include/tasks/fifo/tasks-fifo.h

```diff
@@ -24,11 +24,6 @@
 
 #include "chpl-threads.h"
 
-#ifdef __cplusplus
-extern "C" {
-#endif
```

Comment:
I'm guessing this is a merge error - thanks for pointing it out - I'll un-remove it